commissioning.tex

```diff
@@ -26,9 +26,12 @@ \subsection{Commissioning Plan and Schedule}
 The On-Sky Engineering phase will be carried out with both ComCam and LSSTCam.
 Both the System Optimization and Science Validation (SV) phases will be carried out with LSSTCam. 
 The System Optimization phase will collect a set of observations designed to help optimize the system prior to starting the Science Validation phase.
-During the Science Validation phase, a series of SV Surveys designed to support scientific analyses that validate the system's performance and allow Rubin to demonstrate operations readiness will be carried out. See Section 6 of \citeds{SITCOMTN-005} for the baseline design of the SV surveys.
-The System Optimization and SV phases contain a number of planned key components, including an LSST wide-fast-deep (WFD) 1-2 year equivalent depth ``pilot'' survey.
-In all phases, field selection will be carried out by the commissioning team, taking into account a wide variety of constraints as well as a ``menu'' of science opportunities to which the LSST Science Community has contributed.
+During the Science Validation phase, a series of SV Surveys designed to support scientific analyses that validate the system's performance and allow Rubin to demonstrate operational readiness will be carried out.
+Both the System Optimization and SV phases contain a number of planned key components, which currently include an LSST wide-fast-deep (WFD) 1--2 year equivalent depth ``pilot'' survey and a 
```

Comment:
```suggestion
The System Optimization and SV phases contain a number of planned key components, which currently include an LSST wide-fast-deep (WFD) 1--2 year equivalent depth ``pilot'' survey and a 
```

Reply:
The "both" makes it sound like we'll do a pilot survey in SO and then repeat it in SV...